docs/api-reference/dbos-compiler.md

```diff
@@ -0,0 +1,85 @@
+---
+sidebar_position: 1
+title: DBOS Compiler
+description: DBOS Compiler reference
+---
+
+The DBOS Compiler helps you deploy your DBOS applications with Stored Procedures to PostgreSQL (including your DBOS Cloud database).
+
+## Installation
+
+To install the latest Cloud CLI version for your application, run the following command in your package root:
+
+```
+npm install --save-dev @dbos-inc/dbos-compiler
+```
+
+## Stored Procedure Versioning
+
+Since [@StoredProcedure](../tutorials/stored-proc-tutorial.md) functions must be tied to a specific application version, both DBOS Transact and the DBOS Compiler are version aware. 
+By default, the application version is specified via the `DBOS__APPVERSION` environment variable, but can also be controlled via command line parameters.
+When a application version is specified, the DBOS Compiler will automatically prefix generated stored procedures with `v` and the application version.
+Likewise, DBOS Transact will automatically invoke prefixed versions of the deployed stored procedures when the application version is specified.
+For example, if the application version was `1720214256655`, all of the stored procedures generated by would be prefixed with `v1720214256655_`.
+
+:::warning
+DBOS Cloud cleans up the database deployed stored procedures for older application versions via a background task.
```

Comment:
This doesn't need to be a big scary warning, especially as people are unlikely to try to do this and even if they do, it won't cause unexpected problems.